.travis.yml

```diff
@@ -5,6 +5,7 @@ php:
   - 7.1
```

Comment:
Could you remove PHP 7.1 in the same turn since it's EOL? (also in the `composer.json`)

Reply:
Sure. In next PR I want to add support for PHP 8

Reply:
I'm completely out of the loop on the state of PHP. Are there expected breaking changes that would affect this project?

Reply:
Don't know yet :)